modules/how-to-plan-your-environment-according-to-cluster-maximums.adoc

```diff
@@ -14,7 +14,7 @@ take to avoid memory swapping.
 
 [IMPORTANT]
 ====
-Some of the limits are stretched only in a single dimension. They will vary
+Some of the tested maximums are stretched only in a single dimension. They will vary
 when many objects are running on the cluster.
 ====
 
```

Comment:
Also, in this section can we make clear that the numbers we note on these pages are based on our setup, configuration and tunings and test methodology. Users may therefore find that these numbers are different owing to their own individual setup and environments.